ApplicationDbContext.cs

```diff
@@ -12,15 +22,30 @@ public class ApplicationDbContext: DbContext
         public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
         {
         }
-        //public DbSet<Roles> Roles { get; set; }
-        public DbSet<Address> Address { get; set; }
-        public DbSet<CourseEnrollments> Courses { get; set; }
-        public DbSet<Events> Events { get; set; }
-        public DbSet<MadePayments> MadePayments { get; set; }
-        public DbSet<OutStandingStudentFees> OutStandingStudentFees { get; set; }
-        public DbSet<People> People { get; set; }
-        public DbSet<Roles> Roles { get; set; }
-        public DbSet<StudentCourses> StudentCourses { get; set; }
-        public DbSet<Subjects> Subjects { get; set; }
+        // public DbSet<MadePayments> MadePayments { get; set; }
+        // public DbSet<OutStandingStudentFees> OutStandingStudentFees { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+            modelBuilder.ApplyConfiguration(new RoleSeed());
+            modelBuilder.ApplyConfiguration(new UserSeeding());
+            modelBuilder.ApplyConfiguration(new SubjectSeed());
+            modelBuilder.ApplyConfiguration(new EventSeed());
+            modelBuilder.ApplyConfiguration(new CourseSeed());
+            modelBuilder.ApplyConfiguration(new EnrollmentSeed());
+        }
+        
```

Comment:
Suggestion: try adding query filters to your entities but it does introduce a lot of footguns as well https://learn.microsoft.com/en-us/ef/core/querying/filters